packages/dapp-svelte-wallet/ui/src/Requests.svelte

```diff
@@ -17,19 +15,22 @@
     $: mappedPayments = incomingPayments.map((i) => { 
         return {
             'type': 'payment',
-            'data': i
+            'data': i,
+            'id': i.id,
```

Comment:
Just a minor stylistic thing: for properties that are valid identifiers, please remove the single-quotes.

i.e.
```
    type: 'payment',
    data: i,
    id: i.id,
```

Reply:
Done